galaxy/tools/deps/mulled/util.py

```diff
@@ -123,6 +129,31 @@ def image_name(targets, image_build=None, name_override=None):
         return "mulled-v1-%s%s" % (m.hexdigest(), suffix)
 
 
+def v2_image_name(targets, image_build=None, name_override=None):
```

Comment:
This function really needs a doc test, so that people can see what is hashed into what and can reimplement the hashing.